src/server/api/routes/api-router.js

```diff
@@ -14,6 +14,9 @@ const messagesRouter = require('./messages-router');
 const channelMembersRouter = require('./channel-members.router');
 const notificationsRouter = require('./notifications-router');
 
+// private channel router imports
+const privateChannelRouter = require('./private-channel.router');
```

Comment:
How is this not throwing an error? There is no private channel router right!? You have to check these things wael!